package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "root",
+  "private": true,
+  "husky": {
+    "hooks": {
+      "pre-commit": "bash ./scripts/check-protected-branch",
+      "pre-push": "bash ./scripts/check-protected-branch"
+    }
+  },
+  "devDependencies": {
+    "husky": "^4.2.5"
+  }
```

Comment:
By using husky, we can execute a shell script when we commit and push to the specific branch.